pkg/tsdb/prometheus/utils/utils.go

```diff
@@ -40,3 +40,7 @@ func StartTrace(ctx context.Context, tracer tracing.Tracer, name string, attribu
 		span.End()
 	}
 }
+
+func ToPtr[Type bool | string | int64](v Type) *Type {
+	return &v
+}
```

Comment:
This is just a dummy converter method. I think codegen should provide such functionality. Not for each generated file of course but a more centralized solution. i.e. https://github.com/aws/aws-sdk-go-v2/blob/main/aws/to_ptr.go

Reply:
agreed! 👍

Reply:
provided in #61948